### DISCUSSION TITLE
Spell cooldown timer in triggers?

### DISCUSSION THREAD

**Discussion by tputs** [smasherprog/EqTool#75] [Ideas]
I play wizard. It would be useful to have some cool down timers in the triggers/spells window. Unsure how hard it would be to achieve, but for example I setup custom timers for Harvest in other parse programs to be like "hey dummy use mana skill." Later on we get some spells that have built in cooldowns as well.

Something similar would be dope! Thanks for making this, its my go-to already.

**Comment by smasherprog** [comment]
yup ill add this  good idea

**Comment by smasherprog** [comment]
Just added this it will be available in about 5-10 minutes

**Comment by ethearra** [comment]
Would be great to see this for enchanter's spell Dictate as well - both the cooldown (300s) and the fixed duration on an NPC (48s). Love this feature for Mend on my monk!

**Comment by smasherprog** [comment]
yeah i can add this easily as well. good idea

**Comment by smasherprog** [comment]
This was added
